projects/documentation/content/guides/generating-components.md

```diff
@@ -37,7 +37,18 @@ In response to the prompt above, the package name should be the kebab case versi
    package.json
 ```
 
-From here, peek into the `package.json` file and ensure the "devDependency" of `@spectrum-css/spectrumpattern` listed there is of the most current version. Then, run `yarn` in your terminal to grab any newly-added packages, as well as to ensure that you have the provided CSS processed for use in your component. You'll now be able to see your component in the Storybook, using the command `yarn storybook`, or test its functionality via `yarn test` so that you can dive into fully preparing the pattern for use as a custom element.
+From here, peek into the `package.json` file and ensure the "devDependency" of `@spectrum-css/spectrumpattern` listed there is of the most current version.
+
+Outside of your new package, you will need to manually add information about your new package to both the `tsconfig-all.json` file and the `bundle` package.
+
+The `tsconfig-all.json` config is used to build types for the project in paralell with the JS build that is handled outside of `tsc`. Within that file, find the "references" listing and add an entry for you package (`{ "path": "packages/spectrum-pattern" }`) alphabetically, including a comma at the end of the line if it is not the last entry in the list. This will ensure that the types for your new package are available throughout the library, include at demonstration and test time.
```

Comment:
```suggestion
Open `tsconfig-all.json`, find "references", and add an entry for your package (`{ "path": "packages/component-you-are-adding" }`) alphabetically. The `tsconfig-all.json` config is used to build types for the project in parallel with the JS build that is handled outside of `tsc`. This will ensure that the types for your new package are available throughout the library, include at demonstration and test time.
```